tests/tracer/test_sampler.py

```diff
@@ -94,7 +93,7 @@ def test_sample_rate_deviation(self):
         for sample_rate in [0.1, 0.25, 0.5, 1]:
             tracer = DummyTracer()
 
-            tracer._sampler = RateSampler(sample_rate)
+            tracer._sampler = DatadogSampler(default_sample_rate=sample_rate, rate_limit=-1)
```

Comment:
```suggestion
            # Since RateSampler does not set the sampling priority on a span, we will use a DatadogSampler
            # with rate limiting disabled.
            tracer._sampler = DatadogSampler(default_sample_rate=sample_rate, rate_limit=-1)
```